error_reporting/unit_tests/test_client.py

```diff
@@ -107,9 +107,8 @@ def test_report_exception_wo_gax(self):
         self.assertIsNotNone(target.report_errors_api)
 
     @mock.patch('google.cloud.error_reporting.client.make_report_error_api')
-    def test_report_exception_with_service_version_in_constructor(
-            self,
-            make_client):
+    def test_report_exception_with_service_version_in_constructor(self,
+                                                                  make_client):
```

Comment:
So interestingly this makes everything happy.

```
def test_report_exception_with_service_version_in_constructor(
        self, make_client):
```

8 spaces instead of 4.